anesthetic/plotting/__init__.py

```diff
@@ -1,5 +1,2 @@
-"""Plotting public API.
-
-.. autoclass:: anesthetic.plotting.PlotAccessor
-"""
```

Comment:
I think this is responsible for the issues with building documentation

Reply:
Examples have been added to the docstring for the `pandas2.1` `PlotAccessor`, which is where `pd` is required

Reply:
> Examples have been added to the docstring for the `pandas2.1` `PlotAccessor`, which is where `pd` is required

Removing these lines have fixed the compilation locally, but obviously this isn't viable. I will try removing those lines with the `process_docstring`

I've also noticed there are extra errors happening in the CI I'm not getting locally

Reply:
I've modified the `docs/source/conf.py` to preload pandas such that the `pd` shouldn't be a problem anymore.

> I've also noticed there are extra errors happening in the CI I'm not getting locally

The full command for building the docs matching the CI is:
```bash
make html SPHINXOPTS="-W --keep-going -n"
```
This ensures that we catch whenever we think we are adding external links, but really its not finding anything. Otherwise things pass silently and we are slowly piling on broken links...

Reply:
Ah yes of course, thanks! I'll use this in future